sweagent/frontend/src/components/controls/LRunControl.js

```diff
@@ -1,10 +1,9 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 import { Tab, Tabs } from "react-bootstrap";
 import Form from "react-bootstrap/Form";
-import "../../static/runControl.css";
 import { PlayFill, StopFill } from "react-bootstrap-icons";
-
 import { Link } from "react-router-dom";
+import "../../static/runControl.css";
```

Comment:
These changes are just removing empty lines and placing the CSS import after the components.